dll/shellext/sendmail/lang/bg-BG.rc

```diff
@@ -0,0 +1,6 @@
+LANGUAGE LANG_BULGARIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+{
+    IDS_SHORTCUT, "Shortcut to %s"
```

Comment:
Since this is a new dll, and you do not have a translation for all these languages, you can skip any file that does not have a translation.

Reply:
Deleted except en-US.